test/unit.js

```diff
@@ -26,7 +26,62 @@ describe('bore', () => {
     expect(<Fn />.localName).to.equal('div');
   });
 
-  it('setting attributes', () => {
+  it('setting attributes on CustomElement', () => {
```

Comment:
that numbers suffix confused me I guess, so I didn't pay close attention to them, but my bad. sry nevermind